static/Blog.html

```diff
@@ -0,0 +1,30 @@
+<html>
+<head>
+<title>
+  Just a Ruby Girl, In a Ruby World...</title>
+</head>
+<header>
+  <a href="/home">Home</a>
+  <a href="/about">About</a>
+  <a href="/blog">Blog</a>
+  <a href="/projects">Projects</a>
+</header>
+<body>
+<p>
+Read about my Week 1 experience on my personal blog:<br> <a href="http://misshellymac.com/ada_week_1_retrospective">MisShellyMac.com</a>  <---Click Here!
+  <p>
+
+  Or click on the pic of my adorable dog Cooper!<br>
+  <a href="http://misshellymac.com/ada_week_1_retrospective"> <img border="0" alt = "My Dog Cooper" src="images/cooper.jpg" width = "200" height="200"><br></a>
+<br>
+<p>
+Here's a funny picture just because:<br>
+<img src="images/ha.jpg" alt= ha>
```

Comment:
Missed the quotes around the alt parameter: `<img src="images/ha.jpg" alt="ha">`.